nested_conditionals.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Nested conditionals</title>
+  <meta charset="utf-8">
+  <script>
+    var name = prompt("what is your name?");
+    if (name=="flintstones"){
+      alert("Welcome Pedro flintstones");
+    } else if (name=="Bond") {
```

Comment:
lowercase all letters before doing comparison to prevent case sensitive missmatch